backend/app/models/schemas.py

```
@@ -897,11 +897,46 @@ class RecoveryPositionSummary(BaseModel):
+
+    Sizing cap (issue #234, V1 contract)
+    ------------------------------------
+    The sizing cap is no longer a stored dollar amount — it is a percent of
+    total capital resolved at evaluation time against the connected Schwab
+    account's net-liquidation value. The five sizing-cap fields below mirror
+    the shape the recovery-engine + Settings UI consume:
+
+    - ``sizing_cap_pct`` — whole-percent (e.g. ``25.0``).
+    - ``total_capital`` — Schwab net-liquidation value (``None`` when the
+      account-value cache is unavailable: disconnected / expired / error).
+    - ``resolved_sizing_cap_dollars`` — ``sizing_cap_pct × total_capital ÷ 100``
```

Ruff (0.15.13)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)